src/main/java/service/shop/ShopService.java

```diff
@@ -0,0 +1,8 @@
+package service.shop;
+
+import java.util.List;
+import model.FruitTransaction;
+
+public interface ShopService {
+    void process(List<FruitTransaction> process);
```

Comment:
The method parameter name 'process' should be more descriptive and reflect its purpose. Consider renaming it to something like 'transactions' to improve code readability.